docs/examples/use_cases/pytorch/single_stage_detector/src/data.py

```diff
@@ -62,14 +62,14 @@ def get_train_pytorch_loader(args, num_workers, default_boxes):
 
 def get_train_dali_loader(args, default_boxes, local_seed):
     train_pipe = COCOPipeline(
-        default_boxes, 
+        default_boxes,
         args,
         seed=local_seed)
 
     train_loader = DALIGenericIterator(
-        train_pipe, 
-        ["images", "boxes", "labels"], 
-        118287 / args.N_gpu, 
+        train_pipe,
+        ["images", "boxes", "labels"],
+        reader_name="Reader",
         fill_last_batch=True)
```

Comment:
Apparently SSD doesn't handle well the partial last batch when the reader pads and the accuracy drops significantly. PyTorch dataloader just drops the last, partial batch.